hail/python/hail/methods/relatedness/king.py

```diff
@@ -0,0 +1,289 @@
+    r"""Compute relatedness estimates between individuals using a KING variant.
+
+    .. include:: ../_templates/req_diploid_gt.rst
+
+    Examples
+    --------
+    Estimate the kinship coefficient for every pair of samples.
+
+    >>> kinship = hl.king(dataset.GT)
+
+    Notes
+    -----
+
+    The following presentation summarizes the methods section of Manichaikul,
+    et. al., but adopts a more consistent notation for matrices.
+
+    Let
+
+    - :math:`i` and :math:`j` be two individuals in the dataset
+
+    - :math:`N^{Aa}_{i}` be the number of heterozygote genotypes for individual
+      :math:`i`.
+
+    - :math:`N^{Aa,Aa}_{i,j}` be the number of variants at which a pair of
+      individuals both have heterozygote genotypes.
+
+    - :math:`N^{AA,aa}_{i,j}` be the number of variants at which a pair of
+      individuals have opposing homozygote genotypes.
+
+    - :math:`S_{i,j}` be the set of single-nucleotide variants for which both
+      individuals :math:`i` and :math:`j` have a non-missing genotype.
+
+    - :math:`X_{i,s}` be the genotype score matrix. Homozygous-reference
+      genotypes are represented as 0, heterozygous genotypes are represented as
+      1, and homozygous-alternate genotypes are represented as
+      2. :math:`X_{i,s}` is calculated by invoking
+      :meth:`CallExpression.n_alt_alleles` on the `call_expr`.
+
+    The three counts above, :math:`N^{Aa}`, :math:`N^{Aa,Aa}`, and
+    :math:`N^{AA,aa}`, exclude variants where one or both individuals have
+    missing genotypes.
+
+    In terms of the symbols above, we can define :math:`d`, the genetic distance
+    between two samples. We can interpret :math:`d` as an unnormalized
+    measurement of the genetic material not shared identically-by-descent:
+
+    .. math::
+
+       d_{i,j} = \sum_{s \in S_{i,j}}\left(X_{i,s} - X_{j,s}\right)^2
+
+    In the supplement to Manichaikul, et. al, the authors show how to re-express
+    the genetic distance above in terms of the three counts of hetero- and
+    homozygosity by considering the nine possible configurations of a pair of
+    genotypes:
+
+    +-------------------------------+----------+----------+----------+
+    |:math:`(X_{i,s} - X_{j,s})^2`  |homref    |het       |homalt    |
+    +-------------------------------+----------+----------+----------+
+    |homref                         |0         |1         |4         |
+    +-------------------------------+----------+----------+----------+
+    |het                            |1         |0         |1         |
+    +-------------------------------+----------+----------+----------+
+    |homalt                         |4         |1         |0         |
+    +-------------------------------+----------+----------+----------+
+
+    which leads to this expression for genetic distance:
+
+    .. math::
+
+        d_{i,j} = 4 N^{AA,aa}_{i,j}
+                  + N^{Aa}_{i}
+                  + N^{Aa}_{j}
+                  - 2 N^{Aa,Aa}_{i,j}
```

Comment:
In the images you posted, the tops of the superscripts look cut off

Reply:
So, this is a problem with any symbol that has both subscripts and superscripts. I found the root-cause and proposed a fix to the sphinx-katex people: https://github.com/hagenw/sphinxcontrib-katex/pull/33.

I'm inclined to wait for them to fix rather than try to monkey patch this. I'm not sure that my fix doesn't cause issues on other browsers (though I did verify our website looks fine on a phone).